src/AdoNet/Orleans.Reminders.AdoNet/ReminderService/AdoNetReminderTableOptions.cs

```diff
@@ -17,30 +17,7 @@ public class AdoNetReminderTableOptions
         /// <summary>
         /// Gets or sets the connection string.
         /// </summary>
+        [RedactConnectionString]
```

Comment:
Same as other RedactConnectionString comment. I think we need use Redact, for all formats we've not verified the format of.

Reply:
We explicitly handle SQL connection string in the implementation, but it is true that ADO.NET is generic so I will change it to the plain Redact